tests/src/test/scala/ch/epfl/bluebrain/nexus/tests/kg/files/S3StorageSpec.scala

```diff
@@ -309,4 +311,27 @@ class S3StorageSpec extends StorageSpec {
       } yield assertion
     }
   }
+
+  "Uploading a large file" should {
+    "succeed" ignore {
```

Comment:
Ignored for now as it takes ~2min to complete
We may want to introduce a slow tag for those and still run them from time to time in a scheduled fashion
(there is at least another one for archives iirc)